### PR TITLE
Moves strange seeds from contraband to regular so they no longer have to be paid for.

### PR DESCRIPTION
**Pull request by lilacrhododendron**
:cl:
balance: Strange Seeds are now a regular item in the Megaseed vendor, moved from contraband.
/:cl:

Strange seeds are an important part of botanists' jobs. Making them pay for it just causes a bunch of trouble for botanists and prevents them from actually going out and spending their money on other stuff on the station. The last time someone PRed a similar change was https://github.com/tgstation/tgstation/pull/40848, and they were told by maintainers to simply move necessary items out of contraband rather than making them free, so here it is.

### CODE REVIEW

**Comment by imsxz** [comment]
>be botany >job is to grow food for cook >go on github and make botany buff PR's >tell community its ur job to grow ligger mutation plants to grief with

ok subject

**Comment by Tlaltecuhtli** [comment]
>lets make money more meaningless because

**Comment by Shadowflame909** [comment]
> 
> 
> > lets make money more meaningless because

this system was only meant to hurt and target greytiders

**Comment by subject217** [comment]
Making something more annoying to get doesn't make botanist players stop rushing it. If you have an issue with how botanist players choose to spend their effort you should focus on incentivizing alternatives. This shit is functionally a QOL change for an RNG item. There is no meaningful money interaction from botanists having to pay for this specific item in their machine.

**Comment by 4dplanner** [comment]
"an important part" of their job is probably stretching it a bit, given there will often be shifts where they get nothing useful out of them anyway. Also this is a buff, not just a QOL - being able to buy all 4 strange seeds + nettles roundstart as solo botanist moves your powerspike earlier. Nonetheless, it's not really a sizeable buff, given after the starting few plants you don't really need to buy anything from that point onwards.

**Comment by subject217** [comment]
They were never intended to be economy time gated in the first place, and doing so was an unintended consequence of the economy PR changing all vending machines. Nobody else really cares as much about stuff that's normally in contraband for their vendor.

**Comment by Mickyan** [comment]
You have glossed over the most important argument about this change: why are strange seeds (quoted from ninja's closing comment in the previous PR) "required for normal functioning of the job."?

**Comment by wesoda25** [comment]
Why now? Has something recent prompted you to make this PR? Botany nerfs, or the introduction of economy? Because if the latter, I hardly see how this is any different than what it formally was. Botanists bum rushing tool storage for hacking shit vs dorms for money, or begging an engineer to hack their machine vs begging people for money.

**Comment by Shadowflame909** [comment]
I would like to make a claim for this.

All of the contraband plants, which are a lot, contain the antagonist potential of Botany. It's obvious, it's clear. It's expensive as hell.

Botany gets so little money but has all these expensive plants if they want death nettles, cannabis, fly amanita, or Glowshrooms as an example. It's pretty lame, and not right.


All in All. Goof-code has screwed over Botany's antagonist potential, which leads to the paranoia factor of this game. That is something that should not be accepted.

If all the departments have a way to cause mayhem without unnecessary time gates, why should botany? The mayhem package for Service?

**Comment by BebeYoshi** [comment]
Just fixing what goof spoiled, undocumented changes are bad like nerfing an entire part of a departament. If the nerf to botany was intended it should have been stated in economy's PR/changelog so I don't see a problem with this PR, seems more like a fix than a balance PR though

**Comment by BebeYoshi** [comment]
By "part" of the a departament, I meant cotraband seeds, of course, since they play an important part for botanist that are there to plant more than gaias and food for the cook

**Comment by WaterStBlues** [comment]
Even as a partial botany main, I think this is a bad idea. Adding more strange seeds (even at a higher cost) would be cool tho

**Comment by Tlaltecuhtli** [comment]
botany can sell bananas to clowns if they are out of money

**Comment by FantasticFwoosh** [comment]
Nice powercreep, but it is unessecary when playing roulette with the generated chemicals is made trivial by quantity of unique seeds. Especially when people start becoming converted by felenid mut-toxin tomatoes every other round and other obnoxious or dangerous combinations.

**Comment by BebeYoshi** [comment]
Can't wait to get on the server tonight and give plants to cargo so they can sell it for me and give me money for it and bananas to the clown, surely they won't ignore me since economy is REALLY important, right? hahaha

**Comment by BebeYoshi** [comment]
Also to be honest, it would make more sense if more important seeds like glowshroom were free instead of strange seeds

**Comment by zxaber** [comment]
If it wasn't an issue before goof-economy, why is making them free after such a big deal?

**Comment by BebeYoshi** [comment]
Because people think that economy is something that anyone except Cargo and Botany cares, so they think if you do something for a X person they will give you money for it, in theory of course it's no problem, but if you play the server at least one time a month you will know that it doesn't work like this

**Comment by Shardj** [comment]
> Because people think that economy is something that anyone except Cargo and Botany cares, so they think if you do something for a X person they will give you money for it, in theory of course it's no problem, but if you play the server at least one time a month you will know that it doesn't work like this

Just explicitly ask for money in return, it isn't so hard

**Comment by zxaber** [comment]
You could actually probably just come by Robo and ask me for my money and I'd probably give it to you, since I don't ever have any other use for it.

**Comment by 4dplanner** [comment]
I'm still withholding judgment on this PR, but if botanists are one of the only roles with a real interaction with the economy, that might be more of a problem with the other roles than with botanists.

**Comment by ExcessiveUseOfCobblestone** [comment]
I personally would prefer to see them gone from vendors and into their own crate and then add a modular export for plants that scales with stats/chems so there's incentive for botany to sell the plants and for cargo to process their orders.

**Comment by ExcessiveUseOfCobblestone** [comment]
They're already available in the exotic seed crate for just $1500 per and you get two.

You can actually sell certain seeds right now although I haven't seen it used since $lemon seeds weren't sellable anymore.

**Comment by 4dplanner** [comment]
oh whoops, mb

**Comment by 4dplanner** [comment]
There's already a good way to earn cargo money with botany bounties as well, though

**Comment by Repukan** [comment]
Why are they an important part of botany's job?

How are they an important part of botany's job?

**Comment by zxaber** [comment]
Maybe it's important to the fun of the role for a lot of people? Maybe it gets super boring growing the same wheat every round and it's nice to break up the monotony? 

I dunno, really. I've never played botany.

**Comment by Shadowflame909** [comment]
@Repukan. Antagonist Potential. Antagonist Potential. Xenobiology, mining, atmos, engineering, chemistry, they all have their own unique and destructive antagonist tools. The only one affected by economy in a negative manner, and unintentionally is Botany. All the plants are expensive and botanists don't earn enough.

**Comment by subject217** [comment]
the game isn't just about rolling for antag, and trying to use antags to justify this change is a dumb idea.

**Comment by Shadowflame909** [comment]
@subject217 Subject, I don't see your logic here.

SS13 is about the story of the one station where things went ''wrong'' more or less.

It's about Paranoia and the uncertainty of surviving the events of the station.

As I've said before and I'll keep saying. All departments do, have that one wildcard. With the potential, of causing station-wide mischief.

So I'll keep using this fact to justify that one department should not be unintentionally limited in the amount of mischief it can produce, with means that do not limit all of the departments.

Especially not one of the weakest departments in terms of said mischief.

If these dangerous contraband elements are not to add to the Paranoia and Suspense of ss13? Then what are they there for?

**Comment by granpawalton** [comment]
Really surprised this many people think players roll botanist to grow wheat for the chef

**Comment by Farquaar** [comment]
As someone who plays botany often, I find this change to be wholly unnecessary. Maybe grow some trays for the chef while you wait for your paycheck to come in?

**Comment by granpawalton** [comment]
I mean I play chef and grow my own plants every time I do, if you're on a faster paced round like bagil a paycheck is never fast enough to be viable

**Comment by Shadowflame909** [comment]
@Farquaar  The real issue is that it was an unintentional nerf, and it's a time-gate that no other and potentially more lethal job has.

**Comment by ExcessiveUseOfCobblestone** [comment]
> Really surprised this many people think players roll botanist to grow wheat for the chef

It's the reason why I mained the roll before gene machine introduction with Lady Holmes. It was extremely relaxing to just farm/maintain crops for the chef to make neat dishes with!

**Comment by Shadowflame909** [comment]
I Still grow wheat for the chef. Wheat filled with skeleton mutation toxin

**Comment by imsxz** [comment]
People don't (usually) roll mining to get ores, they do it ti get cool gamer loot or fight mobs, they just get ores because it benefits them and it's expected of them.

People don't (usually) roll RND because they like to AFK and click the button when they get enough points, they do it because station upgrades are insanely strong and they can also get fun memes like the wormhole gun or donksoft vendors.

Botany, without a doubt, is expected to make stuff for chef. Just because it's both uninteresting and not beneficial to the botanist to do this doesn't change that fact. The meme death plnats of botany being accessible without doing what's expected of them is an issue with the job design itself, kinda silly to push changes saying "this is the jobs main purpose because this is how people prefer to play it" kinda as if someone said mining's "job" is to collect loot for themselves.

tl;dr its not botanys job to grow weaponized meme plants

**Comment by zxaber** [comment]
If that's the case, Botany might need a rework to help it mesh better with the needs of the crew. Mining could just fuck off to monster hunter mode as soon as they got the mats for their gear, but the vendors have some neat items that encourage most to at least mine a bit more before heading off to try and gank an ash drake.

Perhaps that would be a fuller solution; some sort of "raw vegetation processor" that turns plants from botany into food items for the chef and grants the botanists "green thumb" points for buying strange seeds with? It seems, if nothing else, a bit lazy to re-use the same format that mining already has.

I still standby the point that it wasn't a majorly discussed issue before goon-economy, and was locked behind a paywall either inadvertently or without any input. Presumably, it's an oversight that should be fixed anyway, and is an inconsistency since every other department was granted free access to their own vendors for gear and supplies.

**Comment by granpawalton** [comment]
Chef has more than enough time to keep food on the table at all times and grow 2-5 plants of his own plants fully modified from the gene tool, i do it all the time and then have time to spare after that. Asking someone to do a job you can do that you can do yourself that they dont want to do is unhealthy and a lot less reliable than doing it yourself.

Theres plenty of other jobs whos sole purpose is gimmicks oriented on primarily helping themselves like genetics, xenobio, etc what is wrong with botany also following suit? Nothing is stopping players from making a meaningful contribution in positive ways either with healing plants, holy water, etc.

If providing cookable food is really something that should be encouraged, the whole trade in point system should be implemented so the unhealthy relationship between chef and botanist can be changed. I dont think it should be strange seeds as the reward though maybe mutant seeds to make upgrading into less useful mutant plants less time consuming while keeping in the standard early time gates and methods for the important ones.

**Comment by ExcessiveUseOfCobblestone** [comment]
Bad elements of the game shouldn't be used as justification for more bad elements of the game tbh.

**Comment by granpawalton** [comment]
I mean Ive never really viewed those as bad elements in the game neither with exotic botany, I do agree with that statement though.

**Comment by FantasticFwoosh** [comment]
#29956 does the OP's suggestive reasoning of easier access to strange seeds (contrary to miner progress following GPS co-ords), but better and loot-based rather than shoehorning it into every round i might add. 

- Not all maps are chef friendly with access doors like box which is divided by a windoor & table, short of dissolving chef & botanist into one 'food/canteen attendant' role.

**Comment by Shadowflame909** [comment]
I don't think people finding fun in meme plants is a bad element of the game. The biogenerator does have fun and overlooked mechanics though

**Comment by ExcessiveUseOfCobblestone** [comment]
See #42590 , Botany/Chem needs to be reworked before PRs which enable more chemical products be introduced.

**Comment by BebeYoshi** [comment]
I don't see the relation between this pr and #42590, one is about adding new chems in plants, which directly gets more chems from chemistry, and this one is just about botany and ecnonomy itself and nothing else, it's actually more related to economy than botany.
